storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -37,7 +46,7 @@ After |xtrabackup| is finished taking the backup you should see the following me
 Preparing the Backup
 ====================
 
-In order to prepare the backup you'll need to specify the keyring-file-data (server-id is stored in :file:`backup-my.cnf` file, so it can be omitted when preparing the backup). 
+In order to prepare the backup you'll need to specify the keyring-file-data (server-id is stored in :file:`backup-my.cnf` file, so it can be omitted when preparing the backup irrelevant to the used |MySQL| version). 
```

Comment:
Instead of irrelevant to the used MySQL version, say

... backup, regardless of the MySQL version used.